src/Function/Properties/Inverse.agda

```diff
@@ -11,15 +11,15 @@ module Function.Properties.Inverse where
 open import Axiom.Extensionality.Propositional using (Extensionality)
 open import Data.Product.Base using (_,_; proj₁; proj₂)
 open import Function.Bundles
-import Function.Properties.RightInverse as RightInverse
+import Function.Properties.RightInverse as RightInverse using (to-from)
 open import Level using (Level; _⊔_)
 open import Relation.Binary.Core using (REL)
 open import Relation.Binary.Bundles using (Setoid)
 open import Relation.Binary.Structures using (IsEquivalence)
 open import Relation.Binary.PropositionalEquality.Core as ≡ using (_≡_)
-import Relation.Binary.PropositionalEquality.Properties as ≡
-import Relation.Binary.Reasoning.Setoid as ≈-Reasoning
+import Relation.Binary.Reasoning.Setoid as ≈-Reasoning using (begin_; _∎)
```

Comment:
I strongly disagree with this. It's not made explicit in any of your other PRs, and is *implicit* in the opening of `Relation.Binary.Reasoning` modules. Any change to the syntax/API for `Reasoning` would need all such uses of `Begin_` etc. to be changed, and I think that's *not* worth the cost. The inconsistency in making the change in this one module seems hard to justify?